style.css

```diff
@@ -618,7 +618,6 @@ html {
     position: fixed;
     width: 80%;
     opacity: 1 !important;
-    height: 85%;
   }
```

Comment:
- [ ] Great job iimplementing the requests from the previous reviewer. However, you unconsciously tampered some of your styles while implementing the requests. At this moment, the `See live` and `See source` are outside the popup card on your desktop. Kindly fix that.

![your-desktop-popup](https://user-images.githubusercontent.com/26597432/224376980-de188468-26e3-40c6-9fea-80b2f86fb435.png)

- [ ] Also, for the mobile behaviou, some part of the popup are not visible as there's no room for scrolling on the popup. Kindly look into this too and fix. Please take a look :point_down: 

[your-mobile-app-behave.webm](https://user-images.githubusercontent.com/26597432/224377529-e6036ce9-829a-44a2-97e7-b57aeb1f5dfa.webm)

Reply:
Fixed.